lib/array.js

```diff
@@ -22,7 +22,10 @@ const shuffle = arr => (
 //   to - <number>, range end
 // Returns: <Array>
 //
-// Example: `range(1, 5) = [1, 2, 3, 4, 5]`
+// Example:
+// range(1, 5)
+// Result:
+// [1, 2, 3, 4, 5]
```

Comment:
Can these at least be on a single line?
```
// Example: range(1, 5)
// Result: [1, 2, 3, 4, 5]
```

Reply:
Oh, sorry, I've done this in other places, but skipped this.